src/actions/users.ts

```diff
@@ -86,3 +88,24 @@ export const unblockUser = (currentUserEmail, blockedUserEmail) => {
     dispatch({ type: 'UNBLOCK_USER', blockedUserEmail })
   }
 }
+
+export const reviewUser = (currentUserEmail, reviewedUserEmail, newStats) => {
```

Comment:
The `reviewUserQuery` does not have a return value that the store should be updated with, and that's why there is only a dispatch to update the exp for `currentUserEmail`